src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -713,7 +713,8 @@ public static int execute(final String[] args) throws Exception {
 
             final boolean legacy = createLegacyMode || (indexExists
```

Comment:
What do you think of creating a (limited) unit test that confirms this behavior?

Reply:
@peternied Added a unit test to verify that a cluster initialized with a V7 security index is not flagged as having a legacy index. The tests works by checking the output of `SecurityAdmin.execute` to ensure that `Legacy index '.opendistro_security' (ES 6) detected (or forced). You should migrate the configuration!` is not found in the output.